<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePage</class>
 <widget class="QWidget" name="HomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>800</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_Home">
   <item>
    <widget class="QScrollArea" name="scrollArea_Home">
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollContent_Home">
      <layout class="QVBoxLayout" name="verticalLayout_HomeContent">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>15</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_Header">
         <item>
          <widget class="QLabel" name="label_Title">
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>18</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>助手2号</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_Header">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnToSettings">
           <property name="text">
            <string>设置</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_Repo">
         <property name="title">
          <string>1. 切换题库 444</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_Repo">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_Repo">
            <item>
             <widget class="QLabel" name="label_Repo">
              <property name="text">
               <string>当前题库:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboRepo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_Parser">
            <item>
             <widget class="QLabel" name="label_Parser">
              <property name="text">
               <string>解析策略:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboParser">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnManageParser">
              <property name="text">
               <string>管理...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_Files">
         <property name="title">
          <string>2. 选择题目文件</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_Files">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_FileTools">
            <item>
             <widget class="QLabel" name="label_Mistake">
              <property name="text">
               <string>错题</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboMistakeOp">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <item>
               <property name="text">
                <string>不启用</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>&gt;=</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>==</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboMistakeCount">
              <property name="currentText">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>次</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="spacer_Files">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnSelectAll">
              <property name="text">
               <string>全选</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layout_SelectTools">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="listWidgetFiles">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>150</height>
             </size>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_Filter">
         <property name="title">
          <string>3. 题型过滤</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_Filter">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_Filter">
            <item>
             <widget class="QCheckBox" name="chkSingle">
              <property name="text">
               <string>单选</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chkMulti">
              <property name="text">
               <string>多选</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chkJudge">
              <property name="text">
               <string>判断</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chkFill">
              <property name="text">
               <string>填空</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="btnPracticeStrategy">
            <property name="text">
             <string>📊 刷题策略...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_Practice">
         <property name="title">
          <string>4. 开始练习</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_Practice">
          <item>
           <widget class="QPushButton" name="btnStartSeq">
            <property name="text">
             <string>顺序练习</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnStartRand">
            <property name="text">
             <string>随机练习</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnViewMode">
            <property name="text">
             <string>看题</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_Exam">
         <property name="title">
          <string>5. 模拟考试</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_Exam">
          <item>
           <widget class="QPushButton" name="btnToExamConfig">
            <property name="text">
             <string>配置并开始考试</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnToHistory">
            <property name="text">
             <string>📜 考试记录</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_Home">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
